taipy/gui/_renderers/factory.py

```diff
@@ -57,6 +57,7 @@ class _Factory:
         "status": "value",
         "table": "data",
         "text": "value",
+        "time": "value",
```

Comment:
from what I read in the component this should `"time": "time"`

Reply:
Thank u very much Fred!